driver/remote/factory.go

```diff
@@ -0,0 +1,111 @@
+package remote
+
+import (
+	"context"
+	"net/url"
+	"path/filepath"
+	"strings"
+
+	"github.com/docker/buildx/driver"
+	dockerclient "github.com/docker/docker/client"
+	"github.com/pkg/errors"
+)
+
+const prioritySupported = 50
+const priorityUnsupported = 90
+
+func init() {
+	driver.Register(&factory{})
+}
+
+type factory struct {
+}
+
+func (*factory) Name() string {
+	return "remote"
+}
+
+func (*factory) Usage() string {
+	return "remote"
+}
+
+func (*factory) Priority(ctx context.Context, api dockerclient.APIClient) int {
+	if api == nil {
```

Comment:
Priority of remote driver should not depend on Docker API.

Reply:
Makes sense, it should just return a fixed priority.

Is there a reason that the kubernetes driver priority depends on the Docker API?